mods/ra/rules/infantry.yaml

```diff
@@ -399,6 +399,8 @@ MEDI:
 		Name: Medic
 	Health:
 		HP: 6000
+	Mobile:
+		Speed: 50
```

Comment:
This won't help. The problem isn't their speed, the problem is that infantry stops to attack an enemy, but medics and mechanics either won't have targets immediately and go on their merry way to their AttackMove location, or have an Attack order converted to a Move order from the start.

The real solution is better micro.